src/components/notebook-utils.js

```diff
@@ -103,7 +104,12 @@ export const getToolsToDisplay = isAzureWorkspace => _.flow(
   _.filter(tool => !!tool.isAzureCompatible === !!isAzureWorkspace)
 )([tools.Jupyter, tools.RStudio, tools.Galaxy, tools.Cromwell, tools.Azure])
 
-const toolToExtensionMap = { [tools.RStudio.label]: tools.RStudio.ext, [tools.Jupyter.label]: tools.Jupyter.ext }
+export const toolToExtensionMap = _.flow(
+  _.map(tool => (tool.ext ? { [tool.label]: tool.ext } : undefined)),
+  _.omitBy(_.isUndefined),
+  _.reduce((accum, curr) => _.merge(curr, accum), {})
+)(tools)
```

Comment:
`_.remove` (which returns an array) is maybe more appropriate than `_.omitBy` (which returns an object). If the order of tools doesn't matter, then the reduce step could be `_.reduce(_.merge, {})`. And `_.merge` handles undefined, so there's no need to filter them out.

So I think this could be written slightly more concisely as:

```suggestion
export const toolToExtensionMap = _.flow(
  _.filter('ext'),
  _.map(tool => ({ [tool.label]: tool.ext }),
  _.reduce(_.merge, {})
)(tools)
```